### PR TITLE
EDU-1294 make user roles clearer

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1294

### CODE REVIEW

File: src/components/admin/user-accounts-tab.yml [hunk 1]
```diff
@@ -90,3 +90,48 @@ userAccountsTab:
     de: |
       Wähle den Speicherplan aus, den du zuweisen möchtest. Keine Auswahl bedeuted
       Löschung {selectedItemCount, plural, =1 {eines im Benutzerkonto vorhandenen Speicherplans} other {von in den Benutzerkonten vorhandenen Speicherplänen}}.
+  roleInfoMarkdown:
+    en: |
+      # User
+
+      Users can create and edit public documents, as long as they are not protected or archived as well as upload public files.
```
Comment (adrianaluchian) [inline src/components/admin/user-accounts-tab.yml:97]: The whole part after the comma sounds a bit strange to me. And they actually can also create and edit private documents too 🤔 
How about "Users can contribute to the platform's content, with the restriction of protected or archived documents."? Hard to come up with something, as you experienced yourself probably.

File: src/components/admin/user-accounts-tab.yml [hunk 2]
```diff
@@ -90,3 +90,48 @@ userAccountsTab:
...
+
+      # Accredited Author
+
+      Accredited authors can create protected public documents with exclusive editing permissions.
+
+      # Maintainer
+
+      Maintainers have extended content management permissions, such as reviewing, approving, protecting or archiving public documents.
```
Comment (adrianaluchian) [inline src/components/admin/user-accounts-tab.yml:105]: "Maintainers have extended content management permissions over public content and can curate documents and the media library resources. For documents these permissions include reviewing, approving, protecting or archiving, as well as hard-deleting sections." would be maybe an alternative?

File: src/components/admin/user-accounts-tab.yml (hunk 1 above)
Comment (ahelmberger) [reply src/components/admin/user-accounts-tab.yml]: Changed according to your suggestions :-)

File: src/components/admin/user-accounts-tab.yml (hunk 2 above)
Comment (ahelmberger) [reply src/components/admin/user-accounts-tab.yml]: Same here.

File: src/components/admin/user-accounts-tab.yml [hunk 3]
```diff
@@ -90,3 +90,48 @@ userAccountsTab:
...
+      They can also hard-delete content such as document sections and files from the media library.
+
+      # Admin
+
+      Admins can configure global settings, manage user accounts including storage plans, as well as execute technical maintenance tasks.
+
+      ***
+
+      *Roles in {appName} are accumulative, meaning that each consecutive role in this list has all the permissions of the
```
[the remark this answers is not in the document]
Comment (ahelmberger) [reply src/components/admin/user-accounts-tab.yml]: I'm very fond of the common tongue, so... changed :laughing:

File: src/components/admin/user-accounts-tab.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/admin/user-accounts-tab.yml]: 🥇 🎉

**Review by adrianaluchian — APPROVED** [review]

**Pull request closed without merging**